### PR TITLE
fix explicted type can be replaced by <>

### PR DESCRIPTION
**Pull request by centralhardware**

### CODE REVIEW

**Comment by glebfox** [comment]
Hi,

Thank you for you contribution. Please remove all changes from **web-toolkit** and **widgets** modules.
* **web-toolkit** - contains Java which is translated to JavaScript code, so it's better not to change it.
* **widgets** - files in the `addons` package contains code from the embedded addons, i.e. this code isn't written by us, so it must remain unchanged.

**Comment by centralhardware** [comment]
fixed

File: modules/web-widgets/src/name/fraser/neil/plaintext/diff_match_patch.java [hunk 1]
```diff
@@ -175,7 +175,7 @@ private LinkedList<Diff> diff_main(String text1, String text2,
         // Check for equality (speedup).
```
Comment (glebfox) [inline modules/web-widgets/src/name/fraser/neil/plaintext/diff_match_patch.java:175]: This file isn't ours. Please revert any changes in this file.

**Review by glebfox — CHANGES_REQUESTED** [review]

**Comment by centralhardware** [comment]
fixed

**Pull request merged**